src/components/TextField.js

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import TextField from '@mui/material/TextField';
+
+export default function BasicTextFields() {
+  return (
+      <TextField id="outlined-basic" label="Outlined" variant="outlined" />
```

Comment:
Not a good ID. Also, you specify this here _and_ everywhere you call it - duplication.

Reply:
Also label should probably not be "outlined".

Reply:
Removed ID as I dont need the identifer on the component.